docs/source/intro/tutorial.md

```diff
@@ -501,22 +501,40 @@ class MultiplyAddWorkChain(WorkChain):
 ```
 
 :::{note}
-Besides WorkChain's, workflows can also be implemented as *work functions*.
+Besides WorkChain's, workflows can also be implemented as [*work functions*](https://aiida.readthedocs.io/projects/aiida-core/en/latest/topics/workflows/concepts.html#topics-workflows-concepts-workfunctions).
 These are ideal for workflows that are not very computationally intensive and can be easily implemented in a Python function.
 :::
 
 Let's run the `WorkChain` above!
+
+:::{note}
+To simplify the basic tutorial, we have left out the steps to register the `MultiplyAddWorkChain` as a plugin which allows us to access the class through the [*entry point*](https://aiida.readthedocs.io/projects/aiida-core/en/latest/topics/plugins.html#topics-plugins-entrypoints) named `core.arithmetic.multiply_add`.
```

Comment:
This is adding the information twice. I would suggest to remove this and mention entry points only after they are used (below)